plugins/browser-plugin-ga-cookies/README.md

```diff
@@ -36,14 +36,26 @@ Initialize your tracker with the GaCookiesPlugin:
 import { newTracker } from '@snowplow/browser-tracker';
 import { GaCookiesPlugin } from '@snowplow/browser-plugin-ga-cookies';
 
-newTracker('sp1', '{{collector}}', { plugins: [ GaCookiesPlugin() ] }); // Also stores reference at module level
+newTracker('sp1', '{{collector}}', { plugins: [ GaCookiesPlugin(
+    /* pluginOptions */
+) ] });
+
+/* 
+ * Available plugin options `GACookiesPluginOptions`:
+ * {
+ * ua: Send Universal Analytics specific cookie values. Defaults to true.
+ * ga4: Send Google Analytics 4 specific cookie values. Defaults to true
+ * ga4MeasurementId: Measurement id/ids to search the Google Analytics 4 session cookie. Can be a single measurement id as a string or an array of measurement id strings. The cookie has the form of _ga_<container-id> where <container-id> is the data stream container id.
```

Comment:
Nit: Maybe `<prefix>_ga_<container-id>`?

Reply:
Tried to keep it the similar to the official documentation but will add a not for the prefix, thanks!